zoomdata/setup.sls

```diff
@@ -0,0 +1,65 @@
+{%- from 'zoomdata/map.jinja' import zoomdata -%}
+
+{%- set url = 'http://localhost:8080' -%}
+{%- set data = [] %}
+{%- set show = {} %}
+
+{%- for user in zoomdata.setup.passwords|default({}, true) %}
+  {%- if zoomdata.setup.passwords[user] == 'random' %}
+    {%- set password = salt['random.get_str'](20) %}
+    {%- if '_' not in password %}
+      {%- set password = password[0] + '_' + password[1:] %}
```

Comment:
so every almost every password will have underscore as 2nd character )

Reply:
only those that do not have underscore in some other place )

Reply:
Exactly!